Products/LDAPMultiPlugins/ActiveDirectoryMultiPlugin.py

```diff
@@ -205,7 +205,13 @@ def getGroupsForPrincipal(self, user, request=None, attr=None):
         filt = '(&(objectClass=%s)(|%s))' % (self.group_class, ''.join(cn_flts))
 
         delegate = acl._delegate
-        R = delegate.search(acl.groups_base, acl.groups_scope, filter=filt)
+
+        if hasattr(acl, 'lookup_groups_base'):
+            lookup_base = acl.lookup_groups_base
+        else:
+            lookup_base = acl.groups_base
+
+        R = delegate.search(lookup_base, acl.groups_scope, filter=filt)
 
         if R['exception']:
             logger.error("Failed to locate groups for principal in %s "
```

Comment:
This message needs to be updated to display `lookup_base` instead of `groups_base`